ui_framework/components/tool_bar/tool_bar_search_box.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render, mount } from 'enzyme';
+import sinon from 'sinon';
+import { commonHtmlProps } from '../../test/common_html_props';
+
+import {
+  KuiToolBarSearchBox,
+} from './tool_bar_search_box';
+
+describe('KuiToolBarSearchBox', () => {
```

Comment:
I like the idea of reducing boiletplate by getting rid of the root describe, but I think we need some structure to our tests to ensure they're written in a consistent style. I think the one rule I'd really really like to keep is one `describe` block per prop. This way you can easily identify which tests relate to which props, and see what kind of assumptions you're making about input (and identify missing use cases).